docs/interactions/structured-messages/webview.md

```diff
@@ -0,0 +1,51 @@
+# Webview
+
+The webview is a convenient way of opening a custom made webpage. It can be chosen by setting the optional target field to `webview` when sending a Structured Message containing a link element, such as [Rich Link](../rich-link), [Carousel](../carousel), or [Template](../template).
```

Comment:
```suggestion
The webview is a convenient way of opening a custom made webpage. It can be chosen by setting the optional `target` field to `webview` when sending a Structured Message containing a link element, such as [Rich Link](../rich-link), [Carousel](../carousel), or [Template](../template).
```